From 8061bcd9c03c26946b6fa3356313e069203f8838 Mon Sep 17 00:00:00 2001
From: Dennis Gilmore <dennis@ausil.us>
Date: Wed, 27 Sep 2023 20:11:40 -0500
Subject: [PATCH] fcc-unlock: add link for new T99W175 5G modem variant

Create link from existing unlock script for the modem in the Lenovo x13s.
IDs are documented at https://cateee.net/lkddb/web-lkddb/MHI_BUS.html

Signed-off-by: Dennis Gilmore <dennis@ausil.us>
---
 data/dispatcher-fcc-unlock/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/dispatcher-fcc-unlock/meson.build b/data/dispatcher-fcc-unlock/meson.build
index 5dc3b6a0..b2a049e9 100644
--- a/data/dispatcher-fcc-unlock/meson.build
+++ b/data/dispatcher-fcc-unlock/meson.build
@@ -25,6 +25,7 @@ install_data(
 
 vidpids = {
   '105b:e0ab': '105b',
+  '105b:e0c3': '105b',
   '03f0:4e1d': '1199',
   '1199:9079': '1199',
   '413c:81a3': '1199',
-- 
2.42.0

